client/ayon_core/hosts/houdini/plugins/publish/collect_karma_rop.py

```diff
@@ -33,6 +33,8 @@ class CollectKarmaROPRenderProducts(pyblish.api.InstancePlugin):
     def process(self, instance):
 
         rop = hou.node(instance.data.get("instance_node"))
+        # to align with maya render layers
+        instance.data["renderlayer"] = rop.name()
```

Comment:
Sorry, what? Why is the ROP node's name used as the `renderlayer`? That seems odd?

Isn't it much easier if a studio wants this for them to us `$OS` as part of the `productName` or `subset` attributes on the node itself so that it's editable by the studio, requires no additional changes?

Reply:
Yes, but then you can't use it in anatomy data. One thing is product name, the other is to use it somewhere else in path. I agree that render layer is somewhat arbitrary name in Houdini context, but most of the DCCs are using render layer is one way or the other and  I wouldn't spam the anatomy data with every DCC specific term.